src/vs/workbench/parts/terminal/electron-browser/terminal.contribution.ts

```diff
@@ -105,6 +105,11 @@ configurationRegistry.registerConfiguration({
 			'type': 'number',
 			'default': 1.2
 		},
+		'terminal.integrated.fontWeight': {
+			'type': 'string',
+			'description': nls.localize('terminal.integrated.fontWeight', "Customizes font weight in terminal."),
```

Comment:
You mean set `terminal.integrated.enableBold` to `false`

Then add class `disable-bold` to provide `font-weight: normal;` right ?